gammapy/catalog/fermi.py

```diff
@@ -340,12 +345,12 @@ def spectrum(self):
 
     def _get_flux_values(self, prefix='Flux', unit='cm-2 s-1'):
         if prefix not in ['Flux', 'Unc_Flux', 'nuFnu']:
-            raise ValueError("Must be one of the following: 'Flux', 'Unc_Flux', 'nuFnu'")
+            raise ValueError(
+                "Must be one of the following: 'Flux', 'Unc_Flux', 'nuFnu'")
```

Comment:
You've added a lot of line breaks, presumably where the length was a little over the 80 char limit.
Did you do this manually or did you editor do this automatically?
(I prefer to be not too strict with the line length, in cases where it's just a little over and less readable when you break.)